tests/functional/test_accounts.py

```diff
@@ -60,6 +60,24 @@ def test_transfer(sender, receiver):
     assert receiver.balance == expected
 
 
+def test_transfer_without_value(sender, receiver):
+    with pytest.raises(AssertionError) as err:
```

Comment:
Use ValueError in these cases.
Raise ValueError when args are not usable. Factor this in to my prior comments about using asserts.

Reply:
ok I got it to work with ValueError